src/client/scripts/esm/game/chess/selection.js

```diff
@@ -39,8 +41,27 @@ import config from '../config.js';
  * including the legal moves it has available.
  */
 
+/**
+ * Bugs:
+ * - Fixed I think?
+ * To do:
+ * - Allow dragging the board with a second finger while dragging a piece.
+ */
+
 /** The currently selected piece, if there is one: `{ type, index, coords }` @type {Piece} */
 let pieceSelected;
+/** If true `pieceSelected` is currently being held. */
+let draggingPiece = false;
+/**
+ * When dropped in the same square, pieces are unselected every second time.
+ * This alows players to move pieces by clicking if they don't want to use drag.
+ * @type{boolean} 
+ * */
+let didLastClickSelectPiece;
+/** Set to false if the user wants to use the original click controls. @type {boolean} */
+let dragEnabled = true;
+/** Is the user using touchscreen. If so we should ignor mouse over. @type {boolean} */
+let touchscreenMode = false;
```

Comment:
docutil.js has a method for determining if the screen is touch-compatible.

Do you know about the chrome dev tools mode that lets you simulate a touch screen?

Reply:
touchscreenMode was a duplicate of pointerIsTouch. I forgot to remove it in commit [aeb699e](https://github.com/Infinite-Chess/infinitechess.org/pull/350/commits/aeb699e201385018e6a8051cb5c9395c46823c58).

> Do you know about the chrome dev tools mode that lets you simulate a touch screen?

No, I didn't. Thank you for the advice! It will be very helpful.